### PR TITLE
bump(main/luv): 1.48.0-1

### LINKED ISSUE #19321: Auto update failing for luv
Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [luv](https://github.com/termux/termux-packages/blob/970362f18c64c8c4a3340db70dbeca77f167e720/packages/luv) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating luv [Current version: 1.47.0-0]
INFO: package being updated to 1.48.0-0.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     9    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404

luv: failed to calculate the new checksum
Already up to date.
ERROR: failed to update checksum.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-02-22 06:13:05 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/8000179030">8000179030</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

### PR DESCRIPTION
**Pull request by Biswa96**
Fixes #19321

### CODE REVIEW

File: packages/luv/build.sh [hunk 1]
```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/luvit/luv
 TERMUX_PKG_DESCRIPTION="Bare libuv bindings for lua"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.47.0-0"
-TERMUX_PKG_SRCURL=https://github.com/luvit/luv/releases/download/$TERMUX_PKG_VERSION/luv-$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=2b5c460fad3b86ea7cfbfaa8bb284506e8f8a354bdd735bde5d664a7b2b1f8bd
+TERMUX_PKG_VERSION="1.48.0-0"
+TERMUX_PKG_SRCURL=https://github.com/luvit/luv/releases/download/v${TERMUX_PKG_VERSION}/luv-v${TERMUX_PKG_VERSION}.tar.gz
```
Comment (thunder-coding) [inline packages/luv/build.sh:6]: This is the first release where I can see the 'v' suffix before version. Might be worth the effort asking the maintaintainers of luv to maintain consistent naming of tarballs

Comment (Biswa96) [reply packages/luv/build.sh:6]: @zhaozg would you like to share any thoughts about the of version style change?

Comment (zhaozg) [reply packages/luv/build.sh:6]: > @zhaozg would you like to share any thoughts about the of version style change?

I made https://github.com/luvit/luv/releases/tag/1.48.0-1

Comment (Biswa96) [reply packages/luv/build.sh]: Thank you. I have updated the tag here.

**Pull request merged**